we-rent-app/src/App.tsx

```diff
@@ -31,26 +33,25 @@ const App = () => {
 	}, []);
 
 	return (
-		<Container>
-			<MyBar sx={{ p: 1, textAlign: "left" }}>
-				<Typography variant="h4" sx={{ p: 1, textAlign: "left" }}>
-					{" "}
-					WERENT
-				</Typography>
-			</MyBar>
-			{listings &&
-				listings.map((listing, i) => (
-					<Typography
-						variant="h4"
-						sx={{ p: 1, textAlign: "center", margin: 10 }}
-						key={i}
-					>
-						{listing.address}, {listing.price}, {listing.num_bedrooms},
-						coordinates: {listing.location.coordinates[0]},{" "}
-						{listing.location.coordinates[1]}, rating: {listing.rating}
+		<>
+			<Container>
+				<MyBar sx={{ p: 1, textAlign: "left" }}>
+					<Typography variant="h4" sx={{ p: 1, textAlign: "left" }}>
+						{" "}
+						WeRent
 					</Typography>
-				))}
-		</Container>
+					<div style={{ position: 'absolute', top: '10px', right: '10px' }}>
```

Comment:
Is it possible to not have the pixel values and instead use non-CSS methods? If it takes too much time, leave it and we can figure for later